deploy-disconnected-registry/olm-sync.sh

```diff
@@ -42,8 +42,14 @@ function prepare_env() {
 function check_registry() {
     REG=${1}
 
+
+
```

Comment:
blank lines___remove please

Reply:
Probably we should start writing some style guidelines like no blank lines, max char per line, etc.

So we have something to share and adhere to while coding, it would make coding and reviewing a lot easier. :)